client/src/templates/Challenges/classic/classic.css

```
@@ -29,6 +29,10 @@
   color: var(--secondary-background);
 }
 
+.light-palette .monaco-editor-tabs [aria-expanded='true'] {
```

The new styles are better, just don't find the need for light palete selector, and the selector will get in the way when adding new theme.


```suggestion
.action-row [aria-expanded='true'] {
  background-color: var(--secondary-color);
}

.monaco-editor-tabs [aria-expanded='true'] {
```

the background can be removed from this selector for clarity

```css
.monaco-editor-tabs [aria-expanded='true'],
.action-row [aria-expanded='true'] {
  border-color: var(--secondary-color);
  color: var(--secondary-background);
}
```